streams-movie-demo/legacy-movies-docker/docker-compose.yml

```diff
@@ -152,45 +155,34 @@ services:
     command: "tail -f /dev/null"
 
   ksqldb-server:
-    image: confluentinc/ksqldb-server:0.6.0
+    image: confluentinc/ksqldb-server:0.9.0
```

Comment:
use 0.10?

Reply:
didn't try 0.10. tho

Reply:
TIL there are some breaking syntax changes in 0.10. I'll stick to 0.9 here :)